lib/systems/examples.nix

```diff
@@ -353,6 +353,7 @@ rec {
 
   wasi32 = {
     config = "wasm32-unknown-wasi";
+    rustc.config = "wasm32-wasi";
```

Comment:
The Rust documentation says "undergoing a [rename to wasm32-wasip1](https://github.com/rust-lang/compiler-team/issues/607)".  Should we use the new name?  Also, should we use the threads variant?

Reply:
That's only available from Rust 1.78, nixpkgs is still on 1.77. Btw, wasm32-wasi will go away January 2025. [cf this timeline](https://blog.rust-lang.org/2024/04/09/updates-to-rusts-wasi-targets.html#timeline)

Reply:
… and 1.78 on master

Reply:
Must have missed that it hit master... Anyway, rebased on master and updated. Thank you.

Problem is that now